### DISCUSSION TITLE
Capstone 23/04 1week

### DISCUSSION THREAD

**Discussion by fkdl0048** [fkdl0048/23_1_Capstone#5] [General]
각자 이번 주 할일 적기

**Comment by fkdl0048** [comment]
- #4 

이번 주 구글 로그인, 플레이 팹 연동 확인

**Reply by fkdl0048** [reply]
이번 주 진행 미스로 이번 주안으로 구현 예정..

**Comment by ThugMS** [comment]
이번주 photon 패키지 import, host 로그인 구현

**Comment by aquamagic9** [comment]
이번주 chatGPT 답변 깔끔하게 나오도록 유도하기. 범용성있게 틀잡기

**Comment by LJI028** [comment]
이번주 캐릭터 이동 구현

**Reply by LJI028** [reply]
터치 시 이동, 카메라가 유저따라다니게 하는거까지 구현 완료!

**Comment by bbabbungtting** [comment]
이번주 타일맵 공부 -> 배경, 맵 구조 구현
포톤 공부
컨셉, 디자인 확정
